### ISSUE TITLE
Gutenboarding: remove or edit unnessessary aria label

### ISSUE THREAD

**Issue by simison** [Automattic/wp-calypso#43554]
In the Gutenboarding canvas we have this:

`aria-label=“Block: WordPress.com onboarding block”`

It's auto-generated because we use editor canvas for the Gutenboarding. It should be removed or edited to make more sense, i.e. describe the onboarding step?
